### PR TITLE
fix: make aws region configurable in core-infra and lb-service

### PR DESCRIPTION
**Pull request by mavogel**
## Description

Added configurable terraform.tfvars for the tf quickstart

## Motivation and Context

I'd like to go through the guide in the readme and deploy the quickstart without errors.

https://github.com/aws-ia/ecs-blueprints/issues/126

## How Has This Been Tested?
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects
- [ ] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

**Comment by bryantbiggs** [comment]
because these are examples, we avoid adding variables since that would move it from being an example to more of a module that is directly consumable. When users want to try out the blueprints, they can clone/fork the project and make any adjustments necessary (including changing the local values to suit their needs). 

tl;dr - modules expose configurable arguments via variables, examples do not

**Comment by mavogel** [comment]
> they can clone/fork the project and make any adjustments necessary (including changing the local values to suit their needs).

Thanks for clarifying :)

**Comment by bryantbiggs** [comment]
this should be resolved now by #128 - thank you!
